pldp/agency/models.py

```diff
@@ -0,0 +1,46 @@
+import uuid
+
+from django.db import models
+from django.utils.translation import ugettext as _
+
+from languages_plus.models import Language
+
+
+class Agency(models.Model):
+    id = models.UUIDField(primary_key=True,
+                          default=uuid.uuid4,
+                          editable=False)
+    name = models.CharField(max_length=1000,
+                            help_text=_("Full name of the agency that is "
+                                        "conducting/posting the study."))
+    department = models.CharField(max_length=1000,
+                                  null=True,
+                                  blank=True,
+                                  help_text=_("Specific department within the "
+                                              "agency that is responsible for "
+                                              "the study. Leave blank, if no "
+                                              "such specification is necessary."))
+    phone = models.CharField(max_length=100,
+                             null=True,
+                             blank=True,
+                             help_text=_("Direct single voice telephone "
+                                         "number for the specified agency."))
+    email = models.EmailField(help_text=_("Single valid email address "
+                                          "actively monitored by the agency's "
+                                          "reception or inquiry desk."))
+    language = models.ForeignKey(Language,
+                                 null=True,
+                                 blank=True,
+                                 on_delete=models.CASCADE,
+                                 help_text=_("Main language used by the "
+                                             "agency posting the study. Leave "
+                                             "blank, if only British English "
+                                             "is used to fill in the study."))
+    type = models.CharField(max_length=255,
```

Comment:
I think this might be an opportunity for another table. It could be pre-populated with the options suggested by the Protocol (p. 10).

```
"governmental agency"
"municipal agency"
"non profit corporation"
"business corporation"
"community organisation"
"educational institute"
"private individual"
```

@beamalsky , here's a question to consider, more generally: to what extent should we diverge from the data modeling in the Protocol? If we do, then we could track those divergences in the README of this repo, maybe?

Reply:
Revising my first comment! 

I think we should just add a [choices property](https://docs.djangoproject.com/en/2.1/ref/models/fields/#choices) to this field, similar to what Eric did elsewhere: https://github.com/datamade/django-pldp/blob/a85d539441d7828d5483baa2d8d017941ae9ee7a/pldp/location/models.py#L165

Reply:
> to what extent should we diverge from the data modeling in the Protocol? If we do, then we could track those divergences in the README of this repo, maybe?

I think this is something we should talk to UCD about sooner rather than later. PLDP is for observational studies, and Just Spaces will need to support intercept surveys out of the box. We're going to be making some shifts and additions to it in any case, but I think there's a strong case to be made for keeping this repo true to the protocol as it's written if we can. Let's discuss on our call, and I'm happy to take a stab at drafting that email.